src/plugins/workspace/public/components/delete_workspace_modal/delete_workspace_modal.tsx

```diff
@@ -43,7 +43,7 @@ export function DeleteWorkspaceModal(props: DeleteWorkspaceModalProps) {
           </ul>
           <EuiSpacer />
           <EuiText color="subdued">
-            To confirm your action, type <b style={{ color: '#000' }}>delete</b>.
+            To confirm your action, type <b>delete</b>.
```

Comment:
Is this change expected?

Reply:
Yes, this looks bad and it's a small change, so include in this PR

![image](https://github.com/ruanyl/OpenSearch-Dashboards/assets/486382/e2f47a49-8226-465b-8640-1e6f643812b7)